Scripts/script-AssignAnalystToIncident.yml

```diff
@@ -27,6 +26,7 @@ script: |
   }
 
   assignBy = args.assignBy || 'random';
+  var onCallArg = true ? args.onCall === 'true' : false;
```

Comment:
Probably should ignore case, no?

Reply:
there are predefined values, so i think its ok